src/physics/tag_types.jl

```diff
@@ -1,4 +1,6 @@
 
+abstract type AbstractTagType end
```

Comment:
Is the abstract type really needed? It doesn't seem to be used anywhere (and `TagType` leaving out the type parameter would probably suffice in any use case).